cmd/kyverno/main.go

```diff
@@ -62,6 +63,7 @@ func main() {
 	flag.StringVar(&excludeGroupRole, "excludeGroupRole", "", "")
 	flag.StringVar(&excludeUsername, "excludeUsername", "", "")
 	flag.IntVar(&webhookTimeout, "webhooktimeout", 3, "timeout for webhook configurations")
+	flag.IntVar(&backgroundScanPeriod, "backgroundScanPeriod", 900, "background scan interval")
```

Comment:
What's the unit here? Can we allow something like `10m`, `1h`, similar to kubectl:
```
      --since=0s: Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs. Only one of
since-time / since may be used.```